### PR TITLE
adding GetGroupMembers to goclock client

### PR DESCRIPTION
**Pull request by agorman**
Thanks for making this great package it really helps me.

I'm hoping you'll add these method to list a group's users.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +158,8 @@ type GoCloak interface {
 	GetUserGroups(accessToken string, realm string, userID string) ([]*UserGroup, error)
 	GetComponents(accessToken string, realm string) ([]*Component, error)
 	GetGroups(accessToken string, realm string, params GetGroupsParams) ([]*Group, error)
-	GetGroup(accessToken string, realm, groupID string) (*Group, error)
+    GetGroup(accessToken string, realm, groupID string) (*Group, error)
```
Comment (SVilgelm) [inline README.md]: please use a \tab symbol here

File: client.go [hunk 2]
```diff
@@ -7,10 +7,8 @@ import (
 	"strings"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
```
Comment (SVilgelm) [inline client.go]: Why did you remove `jwt-go`? It is used by this module, please revert it back

File: client_test.go [hunk 3]
```diff
@@ -1115,6 +1115,40 @@ func TestGocloak_GetGroups(t *testing.T) {
...
+	tearDownGroup, groupID := CreateGroup(t, client)
+	defer tearDownGroup()
+
+	err := client.AddUserToGroup(
+		token.AccessToken,
+		cfg.GoCloak.Realm,
+		userID,
+		groupID,
+	)
+	FailIfErr(t, err, "AddUserToGroup failed")
```
Comment (SVilgelm) [inline client_test.go]: could you use `assert.NoError()` instead of FailIfErr, I', trying to use `assert` library for all new code

File: client_test.go [hunk 4]
```diff
@@ -1115,6 +1115,40 @@ func TestGocloak_GetGroups(t *testing.T) {
...
+
+	users, err := client.GetGroupMembers(
+		token.AccessToken,
+		cfg.GoCloak.Realm,
+		groupID,
+		GetGroupsParams{},
+	)
+	assert.NoError(t, err, "AddUserToGroup failed")
+
+	AssertEquals(
```
Comment (SVilgelm) [inline client_test.go]: and for this one too: `assert.Equal`

File: client_test.go [hunk 5]
```diff
@@ -1115,6 +1115,40 @@ func TestGocloak_GetGroups(t *testing.T) {
...
+		token.AccessToken,
+		cfg.GoCloak.Realm,
+		groupID,
+		GetGroupsParams{},
+	)
+	assert.NoError(t, err, "AddUserToGroup failed")
+
+	assert.Equal(
+		t,
+		len(users),
```
Comment (SVilgelm) [inline client_test.go]: second element is expected, third is actual. Please swap `len(users)` and `1`

Comment (agorman) [reply client_test.go]: Thanks, I should have looked that up.

**Comment by agorman** [comment]
Appreciate the feedback, thanks.

**Comment by SVilgelm** [comment]
@Nerzal you can merge it

**Comment by agorman** [comment]
I made a mistake and should have added this latest commit to a different branch so it wasn't included in this pull request.

It's still something I'd like merged though. How would you like me to proceed?

**Comment by SVilgelm** [comment]
Just do:
```
git reset HEAD^
git checkout -f
git push -f
```

**Comment by agorman** [comment]
Thanks

**Comment by Nerzal** [comment]
LGTM
